bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/ManagedRuleProvider.java

```diff
@@ -24,16 +24,16 @@
  */
 public class ManagedRuleProvider extends AbstractManagedProvider<Rule, String, RuleDTO>implements RuleProvider {
 
-    private RuleManager ruleManager;
+    private RuleEngine ruleEngine;
```

Comment:
Ok, I'll remove RuleRegistry from ManagedRuleProvider and move the implementation logic of add(), update() and remove() methods to the RuleRegistryImpl